### DISCUSSION TITLE
Would it be possible to build on a Chromebook using the Linux Virtual Machine?

### DISCUSSION THREAD

**Discussion by FoxMadeline** [Drewol/unnamed-sdvx-clone#622] [Ideas]
something i was wondering

**Comment by FoxMadeline** [comment]
to clarify; im only interested in doing this for the portability of it, not because i need to

**Comment by 12beesinatrenchcoat** [comment]
It's been a few years (so I don't know it'll still work, or if you're still interested), but yes, you should be able to build this on Chromebook for Linux (at least, I recall doing it, just took a long while). Check https://github.com/Drewol/unnamed-sdvx-clone/blob/develop/build.linux for dependencies you need to install (assuming it's up-to-date).
